content/en/database_monitoring/setup_mongodb/amazon_documentdb.md

```diff
@@ -0,0 +1,200 @@
+"user": "datadog",
+"pwd": "<UNIQUE_PASSWORD>",
+"roles": [
+{ role: "read", db: "admin" },
+{ role: "read", db: "local" },
+{ role: "clusterMonitor", db: "admin" }
+]
+})
+{{< /code-block >}}
+
+Grant additional permissions to the `datadog` user in the databases you want to monitor:
+
+{{< code-block lang="shell" >}}
+db.grantRolesToUser("datadog", [
+{ role: "read", db: "mydatabase" },
+{ role: "read", db: "myotherdatabase" }
+])
+{{< /code-block >}}
+
+Alternatively, you can grant `readAnyDatabase` role to the `datadog` user in the `admin` database to monitor all databases:
+
+{{< code-block lang="shell" >}}
+db.grantRolesToUser("datadog", [
+{ role: "readAnyDatabase", db: "admin" }
+])
+{{< /code-block >}}
+
+{{% /tab %}}
+{{% tab "Sharded Cluster" %}}
+
+1. For each shard in your cluster, connect to the primary node of the shard, create a read-only user for the Datadog Agent in the `admin` database, and grant the required permissions:
+
+{{< code-block lang="shell" >}}
+
+# Authenticate as the admin user.
+
+use admin
+db.auth("admin", "<YOUR_AMAZON_DOCUMENTDB_ADMIN_PASSWORD>")
+
+# Create the user for the Datadog Agent.
+
+db.createUser({
+"user": "datadog",
+"pwd": "<UNIQUE_PASSWORD>",
+"roles": [
+{ role: "read", db: "admin" },
+{ role: "read", db: "local" },
+{ role: "clusterMonitor", db: "admin" }
+]
+})
+{{< /code-block >}}
+
+Grant additional permissions to the `datadog` user in the databases you want to monitor:
+
+{{< code-block lang="shell" >}}
+db.grantRolesToUser("datadog", [
+{ role: "read", db: "mydatabase" },
+{ role: "read", db: "myotherdatabase" }
+])
+{{< /code-block >}}
+
+Alternatively, you can grant the `readAnyDatabase` role to the `datadog` user in the `admin` database to monitor all databases:
+
+{{< code-block lang="shell" >}}
+db.grantRolesToUser("datadog", [
+{ role: "readAnyDatabase", db: "admin" }
+])
+{{< /code-block >}}
+
+2. Follow the same steps and create the same user from a `mongos` proxy. This action creates the local user in the config servers and allows direct connection.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Securely store your password
+
+{{% dbm-secret %}}
+
+### Install and configure the Agent
+
+To monitor your Amazon DocumentDB Cluster, you must install and configure the Datadog Agent on a host that can [remotely access][1] your Amazon DocumentDB Cluster. This host can be a Linux host, a Docker container, or a Kubernetes pod.
+
+<div class="alert alert-info">Amazon DocumentDB Elastic clusters are not supported because they only expose the cluster (mongos) endpoints.</div>
+
+#### Create the configuration file
+
+{{< tabs >}}
+{{% tab "Replica Set" %}}
+{{% dbm-amazon-documentdb-agent-config-replica-set %}}
+{{% /tab %}}
+{{% tab "Sharded Cluster" %}}
+{{% dbm-amazon-documentdb-agent-config-sharded-cluster %}}
+{{% /tab %}}
+{{< /tabs >}}
+
+Complete this section if you have installed the [Amazon DocumentDB integration][3] to enrich instances
+with DocumentDB integration telemetry.
```

Comment:
Could we maybe be a bit more specific as to what this does? For example, "If you installed the Amazon DocumentDB integration to enrich instances
with DocumentDB integration telemetry, do **X**"

Reply:
I rephrased to:

If you installed the [Amazon DocumentDB integration](http://localhost:1313/integrations/amazon_documentdb/) to enrich instances with DocumentDB integration telemetry, add this section to your configuration: